### PR TITLE
Add links to all generated breadcrumbs

### PR DESCRIPTION
**Pull request by fnune**
This fixes an issue whereby generated breadcrumbs would include SEO
microcode that got picked up as invalid by Google due to invalid IDs
being inferred from them.

For full context, see the issue I posted on Docusaurus:
https://github.com/facebook/docusaurus/issues/7241

A way out of this would be to remove the breadcrumbs, but I opted for
adding a generated index page to each category in the sidebar in order
to give every breadcrumb item a link.

image
_The generated pages look like this. Their existence makes the breadcrumb item clickable, and thus valid in the eyes of Google._

### CODE REVIEW

File: docs/mcu/da1469x-sdk-guide.mdx [hunk 1]
```diff
@@ -2,24 +2,16 @@
 id: da1469x-sdk-guide
 title: DA1469x SDK Integration Guide
 sidebar_label: Dialog DA1469x
+description: "Integrate Memfault for Dialog's DA1469x product family."
 ---
 
-import SendingDataOverUdp from "@site/src/pages/_partials/_sending-data-over-udp.mdx";
```
Comment (noahp) [inline docs/mcu/da1469x-sdk-guide.mdx:7]: Intentional changes here?

Comment (fnune) [reply docs/mcu/da1469x-sdk-guide.mdx:7]: Yep these imports were unused.

Comment (fnune) [reply docs/mcu/da1469x-sdk-guide.mdx:7]: I'll double-check that it wasn't just an IDE blip.

Comment (fnune) [reply docs/mcu/da1469x-sdk-guide.mdx:7]: Yup, looks like those were actually just unused imports! Merging.

**Pull request merged**